pkg/kubelet/network/hostport/hostport_manager.go

```diff
@@ -150,6 +159,21 @@ func (hm *hostportManager) Add(id string, podPortMapping *PodPortMapping, natInt
 		// clean up opened host port if encounter any error
 		return utilerrors.NewAggregate([]error{err, hm.closeHostports(hostportMappings)})
 	}
+	isIpv6 := utilconntrack.IsIPv6(podPortMapping.IP)
+
+	// Remove conntrack entries just after adding the new iptables rules. Better to do this here than in Remove because
```

Comment:
I'm find it a little hard to understand this comment. Maybe something like 
```
// Remove conntrack entries just after adding the new iptables rules. If the conntrack entry
// is removed along with the IP tables rule, it can be the case that the packets received by
// the client after IP table rule removal will create a new conntrack entry that will result in
//  blackhole of the traffic.
```